### ISSUE TITLE
Update footer text in all pages

### ISSUE THREAD

**Issue by libregeek** [opencats/OpenCATS#60]
Modify the footer to replace catsone.com url and use opencats.org url. This should be done in login page also.

**Comment by RussH** [comment]
I'd love to.. but we're risking breaching the license. Depends if we consider opencats.org is now the cats project website.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the O2 network.                                                                                                                                                                                                                From: ManilalSent: Saturday, 5 March 2016 03:54To: opencats/OpenCATSReply To: opencats/OpenCATSSubject: [OpenCATS] Update footer text in all pages (#60)Modify the footer to replace catsone.com url and use opencats.org url. This should be done in login page also.

—Reply to this email directly or view it on GitHub.

**Comment by libregeek** [comment]
Yes, we can consider opencats.org as the project site. I'm not sure whether cognizo still has interest in the source code. If you think so, then we can retain it and change it when we rewrite it.

**Comment by RussH** [comment]
I think Cognizo has zero interest in opencats source code, but want to keep the click-thru's. I'd rather wait and change it once all the code is rewritten. Possibly we can ensure it's just a plain text link. The requirement is 'Powered by CATS' with a link to cats.com. Therefore maybe we can move to a text as follows; OpenCATS (linked to opencats.org then 'based upon original work Powered by CATS' with a link to cats.com.

I wouldn't want to enforce this change in the license - folks can remove the opencats.org piece if they need to. Maybe it's too wordy. What do you think?

-- 
Reply sent via Mailstrom! http://mailstrom.co/

**Comment by libregeek** [comment]
+100

**Comment by RussH** [comment]
Closing, see login screen example;
![image](https://user-images.githubusercontent.com/282748/45880109-c4db7800-bd9e-11e8-94d1-1778c6585cda.png)
